presto-main/src/test/java/com/facebook/presto/metadata/TestDiscoveryNodeManager.java

```diff
@@ -153,7 +154,16 @@ public void testGetCoordinators()
     @Test
     public void testGetResourceManagers()
     {
-        DiscoveryNodeManager manager = new DiscoveryNodeManager(selector, nodeInfo, new NoOpFailureDetector(), Optional.empty(), expectedVersion, testHttpClient, new TestingDriftClient<>(), internalCommunicationConfig);
+        class NodeStatusServiceImpl
```

Comment:
Any reason why this isn't a lambda?

Reply:
no reason, let me replace that with lambda.